src/common/options/global.yaml.in

```diff
@@ -5365,6 +5365,30 @@ options:
   desc: Enable health indication when spurious read errors are observed by OSD
   default: true
   with_legacy: true
+- name: bluestore_warn_duration_for_slow_ops
+  type: uint
+  level: advanced
+  desc: Duration of slow op warning of bluestore should persist since last occurence
+  default: 30
+  with_legacy: true
+- name: bluestore_slow_op_event_counter_threshold
+  type: uint
+  level: advanced
+  desc: Last maximum number of slow op event considered
+  default: 1
+  with_legacy: true
+- name: bluestore_warn_duration_for_block_device_stalled_read
+  type: uint
+  level: advanced
+  desc: Duration of stalled read warning of bluestore should persist since last occurence
+  default: 30
+  with_legacy: true
+- name: bluestore_stalled_read_event_counter_threshold
```

Comment:
Generally speaking this is bdev not bluestore parameter, IMO better rename to bdev_stalled_read_warn_threshold